content/html/concepts/attributes/attributes.md

```diff
@@ -13,22 +15,46 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
 ---
 
-Attributes can be added to the opening tag of an HTML element to change its default behavior or provide additional data about it.
 
-Attributes consist of a name and a value in the format `name="value"`:
 
-```pseudo
-<tag name="value"></tag>
+
+# HTML Attributes 
+HTML attributes are essential elements that can be added to the opening tag of an HTML element. These attributes modify the default behavior of the element or provide additional information about it. 
+
+Attributes consist of a name and need to be set off equal to a value. Values are enclosed in double or single quotation marks. 
+## Syntax 
+The syntax for adding attributes to an HTML element is as follows: 
+ ```html 
+ <tagname attribute="value"></tagname> 
+ ```
+ 
+## Examples 
+
+ 1. **Image Attributes**
+
+Consider an `<img>` element, commonly used to display images. It can have several **attributes**, such as `src`, `width`, and `height`, which significantly influence how the image is presented on a web page. As we have declared its dimentions to our element. Hence we control the deserible output of the illustration inside of our HTML file.
+```html
+<img src="source_of_image.png" width="500" height="600" />
```

Comment:
````suggestion
Consider an `<img>` element, commonly used to display images. It can have several attributes, such as `src`, `width`, and `height`, which significantly influence how the image is presented on a web page. 

```html
<img src="source_of_image.png" width="500" height="600" />
````
We don't use first- or second-person pronouns in docs.